### PR TITLE
[stable/sonarqube] Add support of cloudsql proxy.

### PR DESCRIPTION
**Pull request by eugenesenchenko**
Signed-off-by: Eugene Senchenko <senchenkoo@gmail.com>

#### What this PR does / why we need it:
This PR adds support for Google CloudSQL database connections via cloudsql-proxy util

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/sonarqube/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: sonarqube
 description: Sonarqube is an open sourced code quality scanning tool
-version: 3.1.0
+version: 3.1.1
```
Comment (rjkernick) [inline stable/sonarqube/Chart.yaml]: Please update the minor version as this adds new features.

**Review by rjkernick — CHANGES_REQUESTED** [review]

**Comment by rjkernick** [comment]
Please merge in the latest master branch and increment the version in Chart.yaml. Don't forget to sign the commits.

File: stable/sonarqube/Chart.yaml (hunk 1 above)
Comment (eugenesenchenko) [reply stable/sonarqube/Chart.yaml]: Done

**Comment by eugenesenchenko** [comment]
@rjkernick I resolved all conflicts and bumped chart version. Could you please review and approve my PR 
 if you don't have any objections?

**Comment by eugenesenchenko** [comment]
@tsiddique maybe you can review and approve this PR if you don't have any objections on this?

**Comment by eugenesenchenko** [comment]
I resolved conflicts again.
@tsiddique @rjkernick do we need these changes? If not then I can close this PR.

**Comment by davidkarlsen** [comment]
This chart has been deprecated and moved: https://github.com/helm/charts/pull/21007

**Pull request closed without merging**